src/main/java/core/basesyntax/service/impl/OperationServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.exceptions.OperationNotFoundException;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.OperationService;
+
+public class OperationServiceImpl implements OperationService {
```

Comment:
Logic of this class can be a part of `Operation` itself, no need separate service for this.